src/openrct2/rct2/S6Importer.cpp

```diff
@@ -803,7 +803,12 @@ class S6Importer final : public IParkImporter
         dst.ProximityStart = { src.proximity_start_x, src.proximity_start_y, src.proximity_start_z };
         dst.CurrentRide = src.current_ride;
         dst.State = src.state;
-        dst.ProximityTrackType = RCT2TrackTypeToOpenRCT2(src.proximity_track_type, _s6.rides[src.current_ride].type);
+        if (src.proximity_track_type == 0xFF)
```

Comment:
I'm not sure this will work. I think the first thing to check would be if the current ride is invalid.

Reply:
Should I set it to null if `src.proximity_track_type == 0xFF || src.current_ride >= RCT12_MAX_RIDES_IN_PARK`?

Reply:
So I've looked into it and its impossible to get the information we need. We have lost too much information (really we never had the information)  due to the change in track types! We are just going to have to do something hacky and modify the ride rating code to get out of this.

Change the if to the following:
```
if (src.current_ride < RCT12_MAX_RIDES_IN_PARK && _s6.rides[src.current_ride].type != RIDE_TYPE_NULL)
            dst.ProximityTrackType = RCT2TrackTypeToOpenRCT2(src.proximity_track_type, _s6.rides[src.current_ride].type);
        else
            dst.ProximityTrackType = 0xFF;
```
We will then need to modify the ride rating checks L335 and L229 to add an additional hack:
```
// TODO: Hack to be removed with new save format trackType 0xFF should not be here
if (trackType == 0xFF || trackType == TrackElemType::None || ...
```

Reply:
I just thought of something: do we even need to import these at all? Can't we just reset the ratings calc data upon import? 

The only downside I see is that it may take a bit longer for the data to come up if you save and reload during calculation, since it would have to start from scratch.

Reply:
Yes we do need it. Peep decisions are changed depending on rating. Changes to when the rating is available will break the simulation